pom.xml

```diff
@@ -447,6 +447,12 @@
 				<type>jar</type>
 			</dependency>
 			<dependency>
+		    	<groupId>com.tdunning</groupId>
+		    	<artifactId>json</artifactId>
+				<version>1.2-SNAPSHOT</version>
```

Comment:
Hi @bitstorm ,

that's what I mentioned here:

https://github.com/apache/wicket/pull/195#issuecomment-263071173

Sadly the API is not exact the same, but we have to move. Even if we take another json framework we can't provide exact the same methods.

But I hope with open-json it is not that difficult.